msteams-platform/mstdd-landing.yml

```diff
@@ -0,0 +1,94 @@
+### YamlMime:Landing
+
+title: Microsoft Teams Developer Documentation
+summary: Welcome to overview of Microsoft Teams Developer Documentation. Microsoft Teams apps bring key information, common tools, and trusted processes to where people increasingly gather, learn, and work.
+metadata:
+  title: Microsoft Teams Developer Documentation
+  description: Introducing building apps for Teams
+  ms.date: 11/10/2021
+  ms.topic: landing-page
+landingContent:
+- title: Explore new features
+  linkLists:
+    - linkListType: concept
+      links:
+      - text: Monetize your Teams app with a transactable SaaS offer
+        url: ~/concepts/deploy-and-publish/appsource/prepare/include-saas-offer.md
+      - text: Create low-code custom apps
+        url: ~/samples/teams-low-code-solutions.md
+- title: Tutorials
+  linkLists:
+    - linkListType: concept
+      links:
+        - text: Proactive messages
+          url: ~/bots/how-to/conversations/send-proactive-messages.md
+        - text: Add authentication to bots
+          url: ~/bots/how-to/authentication/auth-aad-sso-bots.md
+    - linkListType: tutorial
+      links:
+        - text: Step-by-step guide for Proactive messages
+          url: ~/sbs-send-proactive.yml
+        - text: Step-by-step guide for adding authentication to bots
+          url: ~/sbs-bots-with-sso.yml
+- title: Get started
+  linkLists:
+    - linkListType: get-started
+      links:
+      - text: Get started with building apps
+        url: ../msteams-platform/get-started/get-started-overview.md
+    - linkListType: tutorial
+      links:
+      - text: Build app with JavaScript
+        url: sbs-gs-javascript.yml
+      - text: Build app with Blazor
+        url: sbs-gs-blazor.yml
+      - text: Build app with SPFx
+        url: sbs-gs-spfx.yml
+      - text: Build app with C# or .NET
+        url: sbs-gs-csharp.yml
+      - text: Build app with Node.js
+        url: sbs-gs-nodejs.yml
+- title: About building apps for Teams
```

Comment:
Please change to:

Build apps for Teams

Reply:
fixed